docs/BUILD.md

```diff
@@ -0,0 +1,233 @@
+
+```
+   signingConfigs {
+       release {
+           keyAlias keystoreProperties['keyAlias']
+           keyPassword keystoreProperties['keyPassword']
+           storeFile keystoreProperties['storeFile'] ? file(keystoreProperties['storeFile']) : null
+           storePassword keystoreProperties['storePassword']
+       }
+   }
+   buildTypes {
+       release {
+           signingConfig signingConfigs.release
+       }
+   }
+```
+
+Configure the `signingConfigs` block in your module’s `build.gradle` file.
+
+Release builds of your app will now be signed automatically.
+
+### Building the app for release
+
+These instructions use the App Bundle release format instead of the APK format.
+
+#### Build an app bundle
+
+`cd` into the `client/flutter` directory:
+
+```
+cd .. # this assumes you're in the client/flutter/ios directory, if you're in the app directory you'll want to cd client/flutter
+```
+
+Build the app bundle with flutter:
+
+```
+flutter build appbundle
+```
+
+The release bundle for your app is created at `client/flutter/build/app/outputs/bundle/release/app.aab`.
+
+### Upload the bundle to Google Play
+
+This adapts instructions found [here](https://support.google.com/googleplay/android-developer/answer/7159011).
+
+#### Create a release
+
+A release is a combination of one or more build artifacts that you'll prepare to roll out an app or app update.
+
+To start your release:
+
+1. Go to your [Play Console](https://play.google.com/apps/publish/).
+2. Select an app.
+3. On the left menu, select **Release management > App releases**.
+4. Next to the release type you want to create, select **Manage**, then select **Closed**.
+5. To create a new release, select **Create release**.
+
+#### Prepare your app's release
+
+1. Follow the on-screen instructions to add APKs or app bundles, describe what's new in this release, and name your release. Use the APK generated in [Create a release](#create-a-release).
+2. To save any changes you make to your release, select Save.
+3. When you've finished preparing your release, select Review.
+
+#### Review and roll out
+
+Prerequisite: Before you can roll out your release, make sure you've completed your app's store listing, content rating, & pricing & distribution sections. When each section is complete, you'll see a green check mark next to it on the left menu.
+
+Assets prepared for submission: https://drive.google.com/drive/folders/17wi6q3Vlpt9KB6FuEOpZBdCJHtLCSXzh?usp=sharing
+
+1. Go to your Play Console.
+2. Select an app.
+3. On the left menu, select **Release management > App releases**.
+4. Next to the release you want to roll out, select **Edit release**.
+5. Review your draft release and make any additional changes that are needed.
+6. Select **Review**. You'll be taken to the "Review and roll out release" screen, where you can make sure there aren't any issues with your release before rolling out to users.
+7. Review any warnings or errors.
+8. Select **Confirm rollout**.
```

Comment:
Pretty sure this pushes straight to production.  

> Select Confirm rollout.
If you're rolling out your app's first release on production, clicking Confirm rollout will also publish your app to all Google Play users in the countries you selected.

Reply:
It's a lot harder imagining this in advance rather than actually doing it in the UI... but I'm pretty sure the last time I remember doing this, [this step](https://github.com/WorldHealthOrganization/app/pull/297/files#diff-05ce98cb03d150c05f7863b8b945a9aaR205) is what prevented the prod rollout?

Reply:
I'm dumb :)